### PR TITLE
Fixed CommonNearCacheTestSupport

### PR DESCRIPTION
**Pull request by Donnerbart**
Fixed superfluous creation of `ScheduledExecutorService` in `CommonNearCacheTestSupport`

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/cache/nearcache/CommonNearCacheTestSupport.java [hunk 1]
```diff
@@ -40,20 +40,16 @@ protected NearCacheConfig createNearCacheConfig(String name, InMemoryFormat inMe
     }
 
     protected NearCacheContext createNearCacheContext() {
-        final ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(1);
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/cache/nearcache/CommonNearCacheTestSupport.java]: This is already done in `createNearCacheExecutor()`, which we call to create the `NearCacheExecutor`.

**Comment by Donnerbart** [comment]
run-lab-run

**Comment by Donnerbart** [comment]
run-lab-run

**Comment by tombujok** [comment]
👍

**Pull request merged**